pipelines/SpeciesHabitatIndex/SHI_pipeline.json

```diff
@@ -379,36 +379,36 @@
       "weight": 1
     },
     "data>getAreaOfHabitat.yml@80|country_code": {
-      "description": "Name of the country for the analysis. Check the available countries and regions here: https://www.naturalearthdata.com/downloads/10m-cultural-vectors/",
-      "label": "country",
+      "description": "Name of the country for the analysis. Check the available countries and regions [here](https://www.naturalearthdata.com/downloads/10m-cultural-vectors/).",
```

Comment:
I don't think a link around the word "here" should be our way to link things. Why not something like :

Refer to the [list of available countries and regions on Natural Earth](https://www.naturalearthdata.com/downloads/10m-cultural-vectors/).